gui/next/src/routes/+page.svelte

```diff
@@ -49,7 +50,32 @@ const toggleDescription = what => {
   } else {
     showDescriptionFor = [...showDescriptionFor, what];
   }
-}
+};
+
+
+// purpose:   checks what is the latest pos-cli version in npm registry
+// ------------------------------------------------------------------------
+const getLatestPosCliVersion = async () => {
```

Comment:
on pos-cli start we already check and display a message about a possibility to update

was wondering maybe we could re-use it?

Reply:
I think that one would need some improvements as it often fails for me showing an information that I need to re-run `pos-cli` with `sudo`.